samigo/samigo-app/src/webapp/css/tool_sam.css

```diff
@@ -980,59 +980,70 @@ div.collapsed {
 /*	Tiers for Question Pool Table */
 
 #questionpool .tier0,
-#editform .tier0 {
+#editform .tier0,
+#transferPool .tier0 {
```

Comment:
So what is setting margin-left here that needs an !important override?

Reply:
Not sure, I just noticed that the selector isn't being applied because the form ID doesn't match up.

Reply:
Just did a quick test. Seems like the !important override is not necessary for the Transfer Ownership UI. However, I don't know if it's necessary for the other parts of the selector. So I'm not sure if removing them is safe.

Reply:
Amendment incoming; seems the historical !importants aren't necessary any more for any of the selectors.

Thanks for poking me on that, @ottenhoff.